OfficeDev/microsoft-teams-library-js
# Add Upload File Box for testing upload scenario

## Description

### Main changes in the PR:

1. Added upload File boxAndButton for testing default file upload scenario in webviews/iframe's using `<input type="file">`
2. Added checkbox to enable uploading multiple files
3. Added this JSX element inside FilesAPIs.tsx to reflect a file related scenario.

## Validation

### Validation performed:

1. Load test app and click on "Choose File" under `File Upload` box
2. It should upon up the native file chooser popup.
4. Select any file and choose open to get the file-name reflected in the output section of `File Upload` box

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

<Yes/No> -> No, Its a test app change

### End-to-end tests added:

<Yes/No> -> No,  Its a test app change

## Additional Requirements

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

<Yes/No>, -> No, Its a test app changes
